### PR TITLE
MM-15988 - Making channel menu accessible

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
MM-15988 - Making channel menu accessible

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-15988

### CODE REVIEW

File: components/channel_header/channel_header.js [hunk 1]
```diff
@@ -506,20 +506,25 @@ export default class ChannelHeader extends React.PureComponent {
                     className='channel-header__top'
                 >
                     {toggleFavorite}
-                    <strong
-                        id='channelHeaderTitle'
-                        className='heading'
+                    <button
+                        className='channel-header__trigger style--none'
+                        aria-label={formatMessage({id: 'channel_header.menuAriaLabel', defaultMessage: 'Channel Menu'})}
```
Comment (enahum) [inline components/channel_header/channel_header.js]: @asaadmahmood shouldn't you be using `toLowerCase()` here?

File: components/channel_header/channel_header.js [hunk 2]
```diff
@@ -506,20 +506,25 @@ export default class ChannelHeader extends React.PureComponent {
...
+                        >
+                            <span>
+                                {archivedIcon}
+                                {channelTitle}
+                            </span>
+                        </strong>
+                        <span
+                            id='channelHeaderDropdownIcon'
+                            className='fa fa-angle-down header-dropdown__icon'
+                            aria-label={formatMessage({id: 'generic_icons.dropdown', defaultMessage: 'Dropdown Icon'})}
```
Comment (enahum) [inline components/channel_header/channel_header.js]: same here

File: components/channel_header/channel_header.js (hunk 1 above)
Comment (asaadmahmood) [reply components/channel_header/channel_header.js]: Yup, good point.

**Comment by asaadmahmood** [comment]
Done @enahum

**Pull request merged**